docs/CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+### Changed
+
+### Removed
+
+### Deprecated
+
+## [2.2.0] - 12.04.2021
+### Changed
```

Comment:
we also need to add upgrade Analyzer spacy version to 3.0.5

Reply:
Will add.